modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -211,8 +207,7 @@ public void handleUpdateAll(
                     txId,
                     trackWriteIntent,
                     commitTs,
-                    commitTblId,
-                    commitPartId,
+                    commitPartitionIdAsPartitioned,
```

Comment:
`commitPartitionIdAsPartitioned` is a bit unclear naming.

Reply:
Inlined the variable, it's only used once